### PR TITLE
Correct C icon

### PR DESCRIPTION
**Pull request by mondeja**
c (1)

**Issue:** #3169

### Checklist
  - [ ] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

The icon was not matching with the source, so I've used the current wikimedia version to update it.

### CODE REVIEW

File: .svglint-ignored.json [hunk 1]
```diff
@@ -194,7 +193,7 @@
...
-    "M17.023 9.216s-.386-3.293-3.955-3.385c-3.57-.091-5.402 2.47-5.402 6.13 0 3.66 2.026 6.588 5.498 6.588 3.473 0 3.86-3.66 3.86-3.66l6.656.366s.391 3.306-2.394 5.828C18.5 23.605 15.082 24.017 12.694 24c-2.388-.018-5.698.034-8.9-2.969C.595 18.03.05 15.113.05 12.248c0-2.866.607-6.661 4.414-9.54C7.05.754 9.673.033 12.296.033 23.246.032 23.98 9.28 23.98 9.28z": "C",
+    "M22.393 6c-.167-.29-.398-.543-.652-.69L12.925.22c-.508-.293-1.339-.293-1.847 0L2.26 5.31c-.508.293-.923 1.013-.923 1.6v10.18c0 .294.104.62.271.91.167.29.398.543.652.689l8.816 5.091c.508.293 1.339.293 1.847 0l8.816-5.091c.254-.146.485-.399.652-.689s.271-.616.271-.91V6.91c.002-.294-.102-.62-.269-.91zM12 19.109c-3.92 0-7.109-3.189-7.109-7.109S8.08 4.891 12 4.891a7.133 7.133 0 0 1 6.156 3.552l-3.076 1.781A3.567 3.567 0 0 0 12 8.445c-1.96 0-3.554 1.595-3.554 3.555S10.04 15.555 12 15.555a3.57 3.57 0 0 0 3.08-1.778l3.077 1.78A7.135 7.135 0 0 1 12 19.109zm7.109-6.714h-.79v.79h-.79v-.79h-.79v-.79h.79v-.79h.79v.79h.79v.79zm2.962 0h-.79v.79h-.79v-.79h-.789v-.79h.789v-.79h.79v.79h.79v.79z": "C++",
```
Comment (service-paradis) [inline .svglint-ignored.json:196]: I think this line should not be in this PR

**Review by service-paradis — CHANGES_REQUESTED** [review]
The icon looks different than the icon found in the source:
image

File: .svglint-ignored.json (hunk 1 above)
Comment (mondeja) [reply .svglint-ignored.json:196]: Seems that it's not being marked as conflict.

**Comment by mondeja** [comment]
Now that you mention it, I also see differences with the source, but not as big as the ones you show, are you using the same source as me at https://upload.wikimedia.org/wikipedia/commons/3/35/The_C_Programming_Language_logo.svg?

**Comment by mondeja** [comment]
I've updated it with the source :+1:

**Review by service-paradis — APPROVED** [review]
Perfect! 💯

**Pull request merged**